.travis.yml

```diff
@@ -45,9 +45,20 @@ jobs:
                 - ./dev-tools/vendor/bin/composer-require-checker check composer.json --config-file=.composer-require-checker.json
 
 
+        -
+            stage: Fast Test
+            php: 7.0
+            install:
+                - travis_retry composer update $DEFAULT_COMPOSER_FLAGS $COMPOSER_FLAGS
+                - composer info -D | sort
+            script:
+                - vendor/bin/phpunit || travis_terminate 1
+                - PHP_CS_FIXER_FUTURE_MODE=1 php php-cs-fixer --diff --dry-run -v fix
+
         - &STANDARD_TEST_JOB
             stage: Test
-            php: 7.0
+            php: 5.6
+            env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
```

Comment:
Now all entries that extend `STANDARD_TEST_JOB` will have this environment variable by default.

Reply:
Heh, sorry. Definitely need help for the final version of this PR.

Reply:
Btw what you say doesn't seem to be true:
https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/301348417/config

Reply:
This is because currently all entries that extend `STANDARD_TEST_JOB` override `env`. But if we have an entry without `env` in the future we might not notice that it uses this default.

What do you need help for exactly?

Reply:
Avoiding copy-pasting in YAML can be achieved using references. This is already done with `&STANDARD_TEST_JOB` label and `*STANDARD_TEST_JOB` references in this file.

Out of curiosity: what is wrong with YAML?

Reply:
Source: https://twitter.com/search?q=%40ocramius%20%23yaml&src=typd

1. https://github.com/crdoconnor/strictyaml/blob/master/FAQ.rst#what-is-wrong-with-implicit-typing
1. https://news.ycombinator.com/item?id=12423733

I'll try soon another anchor